source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRShowCardTarget.mm

```diff
@@ -59,23 +59,40 @@ - (IBAction)showCard:(UIButton *)sender
                                                             inputs:inputs
                                                     viewController:_vc
                                                         guideFrame:_superview.frame
-            
+
                                                         hostconfig:_config];
+            unsigned int padding = 0;
+            
+            switch (_config->actions.spacing)
+            {
+                case Spacing::ExtraLarge:
```

Comment:
>case Spacing::ExtraLarge: [](start = 16, length = 25)

You're going to want some `break` statements in here.

Reply:
thanks! 

---
In reply to: [140933853](https://github.com/Microsoft/AdaptiveCards/pull/686#discussion_r140933853) [](ancestors = 140933853)